app/views/log_events/index.html.haml

```diff
@@ -0,0 +1,51 @@
+= content_for :h1 do
+  = text("pages.global_settings")
+
+= content_for :sidebar do
+  = render("admin/shared/sidenav_global", sidenav_tab: "log_events")
+
+%h2= text("pages.event_log")
+
+= form_tag(log_events_path, method: :get) do
+
+  = label_tag(text(:event_filter))
+
+  - option_list = LogEvent::LogEventSearcher::ALLOWED_EVENTS.map { |event| [text(event), event] }
+
+  = select_tag(:events,
+    options_for_select(option_list, params[:events]),
+    multiple: true,
+    class: "js--chosen",
+    data: { placeholder: text(:event_placeholder) })
+
+  = label_tag(text(:start_date))
+  = text_field_tag(:start_date, params[:start_date], class: :datepicker__data)
+
+  = label_tag(text(:end_date))
+  = text_field_tag(:end_date, params[:end_date], class: :datepicker__data)
+
+  = label_tag(text(:query))
```

Comment:
How do you feel about 

![global_settings_-_nucore](https://user-images.githubusercontent.com/5661/28723781-7b7736a2-737d-11e7-85c4-1e2ab8d35416.png)

I wonder if the horizontal spacing needs to be closer?